### DISCUSSION TITLE
Values column

### DISCUSSION THREAD

**Discussion by SandraMarq** [open-sdg/open-sdg#1881] [Q&A]
Hello

How do I change the text of the Values column

![image](https://user-images.githubusercontent.com/118024145/202590635-8ab68550-9f08-4fe0-9038-c9381e34ec0d.png)

Thank you

**Comment by brockfanning** [comment]
@SandraMarq You can control that per-indicator in indicator configuration with [national_geographical_coverage](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#national_geographical_coverage), and you can also control it sitewide in the site configuration [country](https://open-sdg.readthedocs.io/en/latest/configuration/#country).

**Comment by SandraMarq** [comment]
Thank you very much
